### PR TITLE
edits to vignettes; minor spelling corrections

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Since I was trying to understand simulation analysis a bit better, ended up reading all vignettes and had a few edits to submit 😅

### CODE REVIEW

File: vignettes/load-get.Rmd [hunk 1]
```diff
@@ -62,15 +61,16 @@ length(volumeParams)
...
 
-The functions `getAllXXXMatching` can also be used to retrieve entities from multiple paths:
+The functions `getAllContainersMatching()`, `getAllMoleculesMatching()`, and `getAllParametersMatching()` can also be used to retrieve entities from multiple paths:
 
 ```{r getAllEntitiesMatching_multiplePaths}
-#Get the molecule Aciclovir located in `Liver|Periportal|Intracellular` and `VenousBlood|Plasma`
-molecules <- getAllMoleculesMatching(c("Organism|Liver|Periportal|Intracellular|Aciclovir",
-                                       "Organism|VenousBlood|Plasma|Aciclovir"), sim)
+# Get the molecule Aciclovir located in `Liver|Periportal|Intracellular` and `VenousBlood|Plasma`
+molecules <- getAllMoleculesMatching(c("Organism|VenousBlood|Plasma|Aciclovir",
```
Comment (msevestre) [inline vignettes/load-get.Rmd:72]: Interesting... why did you change the order of parameters?

File: vignettes/run-simulation.Rmd [hunk 2]
```diff
@@ -127,9 +130,12 @@ observer <- getQuantity("Organism|Lumen|Aciclovir|Fraction dissolved", sim)
 addOutputs(c(molecule, observer), simulation = sim)
 
 # Add new outputs as path strings
-addOutputs(c("Organism|Lumen|Stomach|Aciclovir", 
-             "Organism|PeripheralVenousBlood|Aciclovir|Whole Blood (Peripheral Venous Blood)"),
-           simulation = sim)
+addOutputs(c(
+  "Organism|Lumen|Stomach|Aciclovir",
```
Comment (msevestre) [inline vignettes/run-simulation.Rmd:134]: Same here. In this case, this is always equivalent. But this is not always the case that changing the order does not matter

**Review by msevestre — COMMENTED** [review]
Looks good to me. Just a few questions but I can merge

File: vignettes/load-get.Rmd (hunk 1 above)
Comment (IndrajeetPatil) [reply vignettes/load-get.Rmd:72]: Because it seemed a bit odd that the second specified pattern appeared first in the output:

image

File: vignettes/run-simulation.Rmd (hunk 2 above)
Comment (IndrajeetPatil) [reply vignettes/run-simulation.Rmd:134]: I _think_ I made this change for the same reason as I above.

File: vignettes/load-get.Rmd (hunk 1 above)
Comment (msevestre) [reply vignettes/load-get.Rmd:72]: Oh I see. Weird
